Header header
float32 segment0
float32 segment1
float32 segment2
float32 segment3
float32 segment4
float32 segment5
float32 segment6
uint8 msg_counter
uint8 sensor_nr
uint8 target_format_type
uint8 segment_index

